backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/queries.html.ftl

```diff
@@ -0,0 +1,322 @@
+    failedNbrElt = document.getElementById("failedNbr");
+    succeedNbrElt = document.getElementById("succeedNbr");
+
+
+}
+
+function updateQueriesTable() {
+    clearTables();
+    queryCounter = 0;
+    updateQueriesInnerCounter();
+    for (i = 0; i < queries.length; i++) {
+        queryCounter++;
+        if(queries[i].status === "RUNNING")
+            runningQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+
+        if(queries[i].status === "NEW")
+            notStartedQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+
+
+        if(queries[i].status === "FAILED")
+            failedQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+
+
+        if(queries[i].status === "DONE")
+            succeedQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+    }
+    if (k <= 30)
+        k++;
+    else k = 0;
+}
+
+function updateQueriesInnerCounter() {
+    runningNbrElt.innerText = queries.filter(x => x.status === "RUNNING").length;
+    notStartedNbrElt.innerText = queries.filter(x => x.status === "NEW").length;
+    failedNbrElt.innerText = queries.filter(x => x.status === "FAILED").length;
+    succeedNbrElt.innerText = queries.filter(x => x.status === "DONE").length;
+}
+
+function clearTables() {
+    for (var i = runningQueriesTable.rows.length - 1; i > 0; i--) {
+        runningQueriesTable.deleteRow(i);
+    }
+
+    for (var i = notStartedQueriesTable.rows.length - 1; i > 0; i--) {
+        notStartedQueriesTable.deleteRow(i);
+    }
+
+    for (var i = failedQueriesTable.rows.length - 1; i > 0; i--) {
+        failedQueriesTable.deleteRow(i);
+    }
+
+
+    for (var i = succeedQueriesTable.rows.length - 1; i > 0; i--) {
+        succeedQueriesTable.deleteRow(i);
+    }
+}
+
+function getQueries() {
+    try {
+        fetch(
+                '/admin/queries', {
+                    method: 'get',
+
+                    headers: {
+                        'Accept': 'application/json'
+                    },
+
+                    credentials: 'same-origin'
+                }).then(response => response.json()).then(data => {
+                queries = data;
+                updateQueriesTable();
+            })
+            .catch(error => {
+                console.log(error);
+            });
+
+    } catch (error) {
+        console.log(error);
+        return null;
+    };
+
+}
+<#noparse >
+    function getHtmltemplate(data) {
+        return `
+
+
+<tr>
+   <td >
+<div class="card container">
+  <div class="card-body">
+      <div class="row">
+         <div class="col title">
+            ${data.label} ( ${data.ownerName} , ${data.id.split('.')[0]} ) : ${data.id.split('.')[1]}
+         </div>
+      </div>
+
+
+
+      <div class="row container">
+         <div class="col inner">
+            Creation-Time : ${((new Date(data.createdAt)).toLocaleString("de-De"))}
```

Comment:
Kannst gerne noch die Locale durchreichen

Reply:
Weißt nicht was Max damit sagen will, aber das  kann evtl echt schiefgehen, wenn die Locale anders gesetzt ist.

Reply:
> Kannst gerne noch die Locale durchreichen

wie meinst du das?

Reply:
Du hast hier "de-De" hardgecoded, da das Template aber aus einen Request-Kontext heraus gerendert wird, kannst du aus dem Request die Locale nehmen `I18n.LOCALE.get()` und sie in dein Template reichen.